src/gen.rs

```diff
@@ -13,24 +13,24 @@ pub struct Cmd {
     r#output: Output,
 }
 
-#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, ArgEnum)]
-pub enum Output {
-    /// Rust trait, client bindings, and test harness
-    Rust,
-    /// Json representation of contract spec types
-    Json,
-}
-
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
     #[error("generate rust from file: {0}")]
-    GenerateRustFromFile(rust::GenerateFromFileError),
+    RustFromFile(rust::GenerateFromFileError),
     #[error("format rust error: {0}")]
     FormatRust(syn::Error),
     #[error("generate json from file: {0}")]
     GenerateJsonFromFile(json::GenerateFromFileError),
```

Comment:
```suggestion
    GenerateRustFromFile(rust::GenerateFromFileError),
    #[error("format rust error: {0}")]
    FormatRust(syn::Error),
    #[error("generate json from file: {0}")]
    GenerateJsonFromFile(json::GenerateFromFileError),
```
Can we keep the Generate prefix for the Rust, or remove the Generate prefix from the Json one? They're the same type of error and benefit from looking the same. It's unclear what benefit we get by shortening one.